Kconfig.zephyr

```diff
@@ -752,7 +752,7 @@ config BUILD_OUTPUT_UF2_FAMILY_ID
 	default "0xada52840" if SOC_NRF52840_QIAA
 	default "0x4fb2d5bd" if SOC_SERIES_IMXRT10XX || SOC_SERIES_IMXRT11XX
 	default "0x2abc77ec" if SOC_SERIES_LPC55XXX
-	default "0xe48bff56" if SOC_SERIES_RP2XXX
+	default "0xe48bff56" if SOC_SERIES_RP2040
```

Comment:
![uf2](https://github.com/user-attachments/assets/222bb09a-2acb-482f-8dd7-87b999ce08a4)
"0xe48bff59" for RP2350

Reply:
I left this out because I haven't demonstrated the `uf2conv.py` produces a valid `.uf2` file that can be loaded onto the target, and I don't want to commit changes that I can't demonstrate work.

Others on Discord have suggested that they have got this working, and I hope they'll be able to share their work. Please remember that this PR is about getting an MVP merged into `main`.